src/main/java/com/maxi/calendar/service/EventServiceImpl.java

```diff
@@ -0,0 +1,68 @@
+package com.maxi.calendar.service;
+
+import java.util.Date;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.maxi.calendar.model.Event;
+import com.maxi.calendar.model.User;
+import com.maxi.calendar.repository.EventRepository;
+import com.maxi.calendar.repository.UserRepository;
+
+@Service
+public class EventServiceImpl implements IEventService {
+	
+	@Autowired
+	EventRepository eventRepository;
```

Comment:
use constructor DI

Reply:
@NoArgsConstructor lombock annotation

Reply:
the previous fix was incorrect, then I change to:
EventServiceImpl (EventRepository eventRepository, UserRepository userReposutory){
		this.eventRepository=eventRepository;
		this.userRepository=userReposutory;
	}